pandas/src/inference.pyx

```diff
@@ -1,5 +1,6 @@
 import sys
 cimport util
+from tslib cimport is_timestamp
```

Comment:
I think I might move this here as well (though may cause a circular import issue).